tests/stack/arithmetic.wast

```diff
@@ -0,0 +1,46 @@
+i32.const 5
+i32.const 7
+i32.add
+#assertTopStack < i32 > 12 "add"
+
+i32.const 5
+i32.const 7
+i32.sub
+#assertTopStack < i32 > 2 "sub"
+
+i32.const 15
+i32.const 3
+i32.mul
+#assertTopStack < i32 > 45 "mul"
+
+i32.const 3
+i32.const 15
+i32.div_u
+#assertTopStack < i32 > 5 "div_u1"
+
+i32.const 2
+i32.const 15
+i32.div_u
+#assertTopStack < i32 > 7 "div_u2"
+
+i32.const 3
+i32.const 15
+i32.rem_u
+#assertTopStack < i32 > 0 "rem_u1"
+
+i32.const 2
+i32.const 15
+i32.rem_u
+#assertTopStack < i32 > 1 "rem_u2"
+
+i32.const 5
+i32.const #pow31 +Int 15
+i32.div_s
+#assertTopStack < i32 > 0 "div_s"
```

Comment:
@pirapira see interaction with `bc` (command line calculator):

```
scale = 4

n = (15 + (2 ^ 31))
d = 5

signed = n - (2 ^ 32)
dived = signed / d
dived

trunced = -429496726
trunced + (2 ^ 32)
```

Produces output:

```
-429496726.6000
3865470570
```

Not sure what's going on with GHCi, but I think that it should be 386547050 (currently asserting 0 in test until this is cleared up).

Reply:
You're right.  GHCi behaves weird.

Sane on small numbers
```
Prelude> -4 `div` 5
0
Prelude> -5 `div` 5
-1
```

Differently on bigger numbers
```
Prelude> (15 + (2 ^ 31) - (2 ^ 32) ) / 5.0
-4.294967266e8
Prelude> (15 + (2 ^ 31) - (2 ^ 32) ) `div` 5
-429496727
```

Reply:
Now i understand what happned: https://ghc.haskell.org/trac/ghc/ticket/643

```
-4 `div` 5
```
is 
```
-(4 `div` 5)
```